src/main/java/com/neatresults/mgnltweaks/json/JsonBuilder.java

```diff
@@ -668,4 +665,21 @@ public void setJson(String json) {
         this.preexisingJson = json;
     }
 
+    // TODO: Move to utils class, e.g. Java8Util
```

Comment:
Argh, I was actively looking in `PropertyUtil` and wondered why this didn't exist. I guess I'm blind. :/